sdk/communication/Azure.Communication.PhoneNumbers/tests/PhoneNumbersClient/PhoneNumbersClientLiveTests.cs

```diff
@@ -1283,6 +1280,30 @@ public void GetOfferingsWithPhoneNumberAndAssignmentType()
             Assert.IsNotNull(offerings);
         }
 
+        [Test]
+        [AsyncOnly]
+        public async Task SearchOperatorInformationAsyncSucceeds()
+        {
+            var number = GetTestPhoneNumber();
+            var client = CreateClient();
+
+            var operatorInformation = await client.SearchOperatorInformationAsync(new List<string>() { number });
+
+            Assert.AreEqual(number, operatorInformation.Value.Values[0].PhoneNumber);
+        }
+
+        [Test]
+        [SyncOnly]
+        public void SearchOperatorInformationSucceeds()
```

Comment:
It would be great if bad request/server error can also be included in unit tests

Reply:
I included them in the async tests, but that's a good call, they should probably be in both places